src/components/BooksForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const bookCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const BooksForm = () => (
```

Comment:
You will have to manage state in this component so you should make it a class component instead of functional. i.e use the `class BookForm extends Component {}` format.